src/foreword.md

```diff
@@ -0,0 +1,90 @@
+<!-- # Foreword
+
+It wasn’t always so clear, but the Rust programming language is fundamentally
+about *empowerment*: no matter what kind of code you are writing now, Rust
+empowers you to reach farther, to program with confidence in a wider variety of
+domains than you did before.
+
+Take, for example, “systems-level” work that deals with low-level details of
+memory management, data representation, and concurrency. Traditionally, this
+realm of programming is seen as arcane, accessible only to a select few who
+have devoted the necessary years learning to avoid its infamous pitfalls. And
+even those who practice it do so with caution, lest their code be open to
+exploits, crashes, or corruption.
+
+Rust breaks down these barriers by eliminating the old pitfalls and providing a
+friendly, polished set of tools to help you along the way. Programmers who need
+to “dip down” into lower-level control can do so with Rust, without taking on
+the customary risk of crashes or security holes, and without having to learn
+the fine points of a fickle toolchain. Better yet, the language is designed to
+guide you naturally towards reliable code that is efficient in terms of speed
+and memory usage. -->
+
+# Prefácio
+
+Não foi sempre tão claro, mas a linguagem de programação Rust é
+fundamentalmente sobre _empoderamento_: não importa que tipo de código você
```

Comment:
Fiquei confuso com uma parada, talvez seja bobo.

Olhando pro comentário acima, a palavra `empowerment` está entre asteríscos, e essa entre underscores, é isso mesmo ou deveriam estar iguais?

Reply:
O efeito prático é o mesmo, que é deixar em itálico. Na implementação do github podemos usar os dois. O meu formatter de markdown no vscode no entanto me força usar underlines...

Reply:
Beleza!